contracts/base/ModuleManager.sol

```diff
@@ -6,9 +6,10 @@ import "./Executor.sol";
 
 /**
  * @title Module Manager - A contract managing Safe modules
- * @notice Modules are pluggable extensions to the Safe that can be added to the Safe by the owner.
-           Modules are a security risk since they can execute arbitrary transactions, so only trusted
-           and audited modules should be added to the Safe. 
+ * @notice Modules are omnipotent extensions to a Safe that can be added to a Safe by its owners.
+           ⚠️ WARNING: Modules entail a security risk since they can execute arbitrary transactions, 
```

Comment:
I'd say the same for "enatil".

```suggestion
           ⚠️ WARNING: Modules are a security risk since they can execute arbitrary transactions, 
```